apps/frontend/src/app/pages/company/applications-job/applications-job.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="container mx-auto min-h-screen px-3">
+  <div class="pb-4 pt-6 flex justify-between flex-wrap gap-4">
+    <div
+      class="flex gap-2 items-center py-4 cursor-pointer"
+      [routerLink]="['/']"
+    >
+      <i class="bi bi-arrow-left text-base-100 text-xl"></i>
+      <p class="font-bold text-sm text-base-100 font-mono">Back</p>
+    </div>
+    <label class="input flex items-center gap-2 bg-transparent text-base-200">
+      <i class="bi bi-search"></i>
+      <input
+        type="text"
```

Comment:
Shouldn't it be a component? We have a lot of pages that we use the search, so the best practice is to componentize it.